plugin/builtin/patchstrategicmergetransformer/PatchStrategicMergeTransformer_test.go

```diff
@@ -8,9 +8,31 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/kustomize/api/konfig"
 	kusttest_test "sigs.k8s.io/kustomize/api/testutils/kusttest"
 )
 
+func condExpected(s1, s2 string) string {
```

Comment:
This function name is pretty hard to understand from the perspective of caller. Maybe add a comment?